WPFClient/Pages/Login.xaml.cs

```diff
@@ -9,9 +10,28 @@ namespace WPFClient
 {
     public partial class Login : Page
     {
+        public static string Status { get; set; } = string.Empty;
         public Login()
         {
+            statusLabel.Content = Status;
             InitializeComponent();
+
+            new Thread(RecieveConnectionStatus).Start();
+        }
+
+        private void RecieveConnectionStatus()
+        {
+            App._connection.On("ReceiveConnectionStatus", (string status) =>
```

Comment:
So I need to invoke this method from the server? is that the intention?

Reply:
I tried already. That's one of the additions I made to the server. I wanted to see if the server could manually change the connection status and push it to the clients, but no luck. So then I tried just sending the connection ID to see if I could compare it to the client's id.. I don't think the ID is even available client side though